### PR TITLE
feat: Add operational admin functions (SC-14970)

### PR DESCRIPTION
**Pull request by Preethisenthilkumar**

### CODE REVIEW

File: technical-resources/admin-functions/operational-admin-actions.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Overview
+
+The operationalAdmin complements the Governor's capabilities by having the authority to execute essential operational functions for routine protocol management. The role holds limited powers compared to the Governor, ensuring a balance between operational efficiency and security.
```
Comment (vbidin) [inline technical-resources/admin-functions/operational-admin-actions.md:3]: nitpick: Rename to 'operational admin` or `operationalAdmin` (in code quotes)

File: technical-resources/admin-functions/operational-admin-actions.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+* `setValidPoolDelegate`
+* `setValidBorrower`
+
+## Pool Manager
+
+* `finishCollateralLiquidation`
+* `setPendingPoolDelegate`
+* `triggerDefault`
+
+## Withdrawal Manager
```
Comment (vbidin) [inline technical-resources/admin-functions/operational-admin-actions.md:29]: Specify it's the cyclical withdrawal manager and add a section for the queued one as well.

**Review by vbidin — CHANGES_REQUESTED** [review]

File: technical-resources/admin-functions/operational-admin-actions.md (hunk 2 above)
Comment (Preethisenthilkumar) [reply technical-resources/admin-functions/operational-admin-actions.md:29]: added ppm too, should we update wiki Globals Update ‐ Q4 Release ?

**Review by JGcarv — APPROVED** [review]

File: technical-resources/admin-functions/operational-admin-actions.md (hunk 2 above)
Comment (JGcarv) [reply technical-resources/admin-functions/operational-admin-actions.md]: No need. The wiki will be deleted after the release

**Pull request merged**